README.md

```diff
@@ -58,5 +41,118 @@ You can also track upcoming changes using the [awaiting-deployment](https://gith
+GA the image must meet the following criteria: 1- Has been through a Beta period (public or private) 2- Most major software we install on the image has a compatible
+version for the underlying OS and 3- All major bugs reported during the Beta period have been addressed. This image type falls under the customer [SLA](https://github.com/customer-terms/github-online-services-sla) for actions. GA images are eventually deprecated according to our guidelines as we only support the
+latest 2 versions of an OS.
+
+## Image Releases
+*How to best follow along with changes*
+
+1. Find the latest releases for this repository [here.](https://github.com/actions/virtual-environments/releases)
+2. Subscribe to the releases coming out of this repository, instructions [here.](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#configuring-your-watch-settings-for-an-individual-repository)
+3. Upcoming changes: A pre-release is created when deployment is started. As soon as deployment is finished, it is converted to a release. If you have subscribed to releases, you will get notified of pre-releases as well.
+   - You can also track upcoming changes using the [awaiting-deployment](https://github.com/actions/virtual-environments/labels/awaiting-deployment) label.
+4. For high impact changes, we will post these in advance to the GitHub Changelog on our [blog](https://github.blog/changelog/) and on [twitter](https://twitter.com/GHchangelog).
+   - Ex: breaking changes, GA or deprecation of images
+5. You can also view notable upcoming changes by viewing the [Announcement](https://github.com/actions/virtual-environments/labels/Announcement) label.
+
+*Cadence*
+
+- We typically deploy weekly updates to the software on the runner images.
+
+## Software and Image Support
+
+### Support Policy
+- Tools and versions will typically be removed 6 months after they are deprecated or have reached end-of-life
+- We support(at maximum) 2 GA images and 1 beta image at a time. We begin the deprecation process of the oldest image label once the newest OS image label has been released to GA.
+- The images generally contain the latest versions of packages installed except for Ubuntu LTS where we rely on the Canonical-provided repositories mostly.
+- Popular tools can have several versions installed side-by-side with the following strategy:
+
+| Tool name | Installation strategy |
+|-----------|-----------------------|
+| Docker images | not more than 3 latest LTS OS\tool versions. New images or new versions of current images are added using the standard tool request process |
+| Java      | all LTS versions |
+| Node.js   | 3 latest LTS versions |
+| Go        | 3 latest minor versions |
+| Python <br/> Ruby | 5 most popular `major.minor` versions |

```

Comment:
Wonder if we should revisit this one since the rest follow 2-3 versions

Reply:
good point - will add as agenda item for our next image sync

Reply:
For Ruby and Python it makes sense to have 5 versions as all of them usually widely used

Reply:
considering we have [setup-python](https://github.com/actions/setup-python) and ruby itself maintains their own [setup-action](https://github.com/ruby/setup-ruby) that can install different versions... seems like we could support the latest 3 and let users install their own if they want something different. this helps keeps things consistent, simple, + small bonus of reduced image size

Reply:
@samvantran I'm afraid the https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/use-python-version?view=azure-devops AzDO task is not able to install python on the fly, the only option is to use the pre-cached versions. The same is for the ruby task https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/use-ruby-version?view=azure-devops